src/Components/EnergyCalculator/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,75 @@
+import { FormattedMessage } from 'react-intl';
+import QuestionHeader from '../../QuestionComponents/QuestionHeader';
+import QuestionQuestion from '../../QuestionComponents/QuestionQuestion';
+import { useEffect } from 'react';
+import { OTHER_PAGE_TITLES } from '../../../Assets/pageTitleTags';
+import { ReactComponent as Apartment } from '../Icons/Apartment.svg';
+import { ReactComponent as Housing } from '../../../Assets/icons/residence.svg';
+import { CardActionArea, Card, Stack, CardContent, Box } from '@mui/material';
+import './LandingPage.css';
+
+const LandingPage = () => {
+  useEffect(() => {
+    document.title = OTHER_PAGE_TITLES.energyCalculatorLandingPage;
+  }, []);
+
+  return (
+    <div className="eCalc-container">
+      <div className="eCalc-landingPage.header">
```

Comment:
Class names should be in kebab-case and should not contain any dots. Dots are used in the CSS selector to represent that you are looking for a class.
```suggestion
      <div className="energy-calculator-landing-page-header">
```